.travis.yml

```diff
@@ -43,7 +43,7 @@ before_script:
   - ./configure
 
 script:
-  - make -j2
+  - make -j$(sysctl -n hw.ncpu)
```

Comment:
@xunzhang ,  could you tell what's the meaning for this parameter "hw.ncpu"? Have you uploaded the code to HAWQ master code? Thanks